migration/sql-files/013-add-state-to-auth-state-reference.sql

```diff
@@ -1,2 +1,4 @@
 -- Alter Oauth state reference table to add state used in oauth workflow
-ALTER TABLE oauth_state_references ADD COLUMN state TEXT unique;
\ No newline at end of file
+ALTER TABLE oauth_state_references ADD COLUMN state TEXT unique;
+UPDATE oauth_state_references SET state = ID;
+ALTER TABLE oauth_state_references ALTER state SET NOT NULL;
```

Comment:
Use `ALTER TABLE oauth_state_references ALTER state SET NOT NULL CHECK (state <> '')` to make sure state is not an empty string.